config/filter.d/nginx-http-errors.conf

```diff
@@ -0,0 +1,7 @@
+# Nginx client and server errors (HTTP 4xx or 5xx)
+#
+# Author: David Cook
+# Inspired by https://serverfault.com/a/849859/959055
```

Comment:
Don't write whereby it's inspired, write what this filter is doing and which issues one can expect by activating of such jails (false positives and banning of legitimate users for instance).